src/storage/mod.rs

```diff
@@ -570,7 +570,7 @@ impl<F: Field> Circuit<F> for EthBlockStorageCircuit<F> {
                                     .iter()
                                     .flat_map(|(slot, value)| slot.iter().chain(value.iter())),
                             )
-                            .map(|acell| acell.cell().clone()),
+                            .map(|acell| *acell.cell()),
```

Comment:
Unfortunately this clippy needs to stay unfixed because if feature "halo2-pse" is used, then `acell.cell()` returns a `Cell` instead of `&Cell` and the compiler will complain you cannot dereference. Using `clone` was our lazy way to get around this without doing feature-specific stuff.